contrib/cirrus/runner.sh

```diff
@@ -338,6 +338,15 @@ function _run_gitlab() {
     return $ret
 }
 
+function _run_fcos_image_build() {
+    if [[ $CIRRUS_PR == '' && $CIRRUS_BRANCH == 'main' ]]; then
+        # wait-for-copr attempts 180 tries with 10s between each try by default
+        wait-for-copr --owner rhcontainerbot --project podman-next podman `git rev-parse --short HEAD`
+        podman build -t quay.io/podman/fcos:podman-next -f ./contrib/podman-next/fcos-podmanimage/Containerfile .
+        podman push quay.io/podman/fcos:podman-next
+    fi
```

Comment:
IIRC, previously we discussed doing something like...

```suggestion
    if [[ "$CIRRUS_PR" == '' ]]; then
        # wait-for-copr attempts 180 tries with 10s between each try by default
        wait-for-copr --owner rhcontainerbot --project podman-next podman `git rev-parse --short HEAD`
    fi
    
    podman build -t quay.io/podman/fcos:podman-next -f ./contrib/podman-next/fcos-podmanimage/Containerfile .

    if [[ "$CIRRUS_PR" == '' ]]; then
        req_env_vars PODMAN_USERNAME PODMAN_PASSWORD
        (set +x; echo "$PODMAN_PASSWORD" | \
            podman login --username "$PODMAN_USERNAME" --password-stdin \
            "quay.io/podman")
        podman push quay.io/podman/fcos:podman-next
    fi
```